composer.json

```diff
@@ -16,16 +16,16 @@
         "nesbot/carbon": "^1.39 || ^2.22",
         "dragonmantank/cron-expression": "^2.3 || ^3.0",
         "ramsey/uuid": "^3.9 || ^4.0",
-        "symfony/cache": "^4.4 || ^5.1.5",
-        "symfony/console": "^4.4 || ^5.1.5",
-        "symfony/dotenv": "^4.4 || ^5.1.5",
-        "symfony/expression-language": "^4.4 || ^5.1.5",
-        "symfony/filesystem": "^4.4 || ^5.1.5",
-        "symfony/finder": "^4.4 || ^5.1.5",
-        "symfony/http-client": "^4.4 || ^5.1.5",
-        "symfony/process": "^4.4 || ^5.1.5",
-        "symfony/string": "^4.4 || ^5.1.5",
-        "symfony/uid": "^4.4 || ^5.1.5",
+        "symfony/cache": "^4.4 || ^5.1.5 || 6.0.*",
```

Comment:
We were discussing replacing `ref` with `service` with @natepage (after #825) and decided to postpone this for the next major release. I think we can change the constraints in that major release as well.

Reply:
Yes, we can. But the point is that we already use `service`. I.e. we declare that we support the 4th version but in fact, we don't.

Reply:
As I see we use the `service` function only in the `eonx-com/easy-activity` for now. So we can't drop support of the 4th version that easy. So we need to introduce the 4th version of `easy-monorepo`.

Reply:
In EasyAdmin too.

Reply:
OK, that means we should use `ref` instead of `service` in EasyActivity. Then everything will be consistent. And we'll get rid of Symfony4 support in the next major monorepo release.

Reply:
I've changed all the `service` usages to `ref`